opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         self.value_type = value_type
         self.enabled = enabled
         self.monotonic = monotonic
+        self.last_update_timestamp = ns_to_iso_str(time_ns())
```

Comment:
I thought we want to use nanoseconds since epoch as the time format across the SDK, and do the conversion in the exporter?